src/python/cli_new/lib/cli/plugins/task/main.py

```diff
@@ -143,3 +148,25 @@ def list(self, argv):
                                .format(error=exception))
 
         print(str(table))
+
+    def inspect(self, argv):
+        """
+        Show the low-level information of the task.
+        """
+        try:
+            master = self.config.master()
+        except Exception as exception:
+            raise CLIException("Unable to get leading master address: {error}"
+                               .format(error=exception))
+
+        data = get_tasks(master, self.config)
+        try:
+            for task in data:
+                if task["id"] != argv["<task_id>"]:
+                    continue
+
+                print(json.dumps(task, indent=4))
+
+        except Exception as exception:
```

Comment:
Can anything actually throw here?
I think it'd be fine to remove the except clause.

Reply:
U are right, should not throw failures. :-)